src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,25 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer volkswagen = manufacturerDao
+                .create(new Manufacturer("Volkswagen", "Germany"));
+        Manufacturer skoda = manufacturerDao
+                .create(new Manufacturer("Skoda", "Czech Republic"));
+        Manufacturer hyundai = manufacturerDao
+                .create(new Manufacturer("Hyundai", "Republic of Korea"));
+        manufacturerDao.getAll().forEach(System.out::println);
+        hyundai.setCountry("Korea");
+        manufacturerDao.update(hyundai);
+        System.out.println(manufacturerDao.get(3L));
```

Comment:
```suggestion
        System.out.println(hundai.getId());
```

Better not to get attached to a certain id, we can get it from Manufacturer entity